# Portfolio
Here I summarize the projects I am actively working on or have worked on in the past

## Active projects
* [High throughput micorscopy dashboard](https://github.com/chuvalab/dashboard): A web app to visualize marker intensities from a table generated from microscopic images. Written in python, makes use of dash and pandas libraries and can be deployed with Docker
* [Single-cell dataset processing workflow with Deep Learning](https://github.com/chuvalab/pgclc_scvi): Use SCVI, a deep learning model based on the Autoencoder archtecture to build a single-cell RNA-seq processing workflow in Python. 
* [Spatial transcriptomics workflow](https://github.com/chuvalab/spatial_transcriptomics_exploring/): A jupyter notebook workflow on Xenium spatial transcriptomics dataset using Python package squidpy for a quick analysis
* Metabolomics imaging dataset (MALDI-MSI) analysis. Available on gitlab.
* [Contributing to web app for education](https://github.com/KnowledgePuzzle). This is a collaborative side project and the repo is currently private. Hands on experience with running open source LLM models, building a RAG pipeline. Exposed to django, html, javscript.



## Past (non-active) Projects
* [Differential methylation analysis tool](https://github.com/johnmous/methylation): A command line tool for the anlysis of amplicon methylation data. Outputs tables and plots
* [Material for a workshop on single-cell RNA-seq data analysis with R](https://github.com/chuvalab/sc_analysis_workshop): Material used during a single-cell analysis workshop aimed at biologists with minimal to none programing experience
* Several single-cell analysis workflows, whose output is part a scientific publication. Some examples [here](https://github.com/chuvalab/embryo_gonads), [here](https://github.com/chuvalab/fibrosis) and [here](https://github.com/johnmous/single_cell-embryo_gonads)
* [Contributing to ChIP-seq analysis pipeline](https://github.com/biowdl/ChIP-seq), writen in WDL
